### PR TITLE
Add visual basic snippet code

### LINKED ISSUE #2596: Add VB version - HttpClient class
The following topic is missing the VB sample:
https://docs.microsoft.com/en-us/dotnet/api/system.net.http.httpclient

### PR DESCRIPTION
**Pull request by Youssef1313**
Related to:
https://github.com/dotnet/samples/pull/1059

Fixes #2596

### CODE REVIEW

**Comment by BillWagner** [comment]
closing and reopening to check build status.

**Comment by BillWagner** [comment]
dotnet/samples#1059 has been merged. Removing the "hold" label.

**Comment by mairaw** [comment]
I kicked off the build for the master branch to pick up the latest samples and will restart this build once that's done.

**Comment by Youssef1313** [comment]
> 
> 
> VB example now displays nicely in the Examples section. There's still an example in C# only in the remarks for the HttpClient class (line 61). Can you also address that @Youssef1313? Thanks!

Thanks, I've added it.

File: xml/System.Net.Http/HttpClient.xml [hunk 1]
```diff
@@ -166,8 +166,19 @@ public class GoodController : ApiController
         HttpClient = new HttpClient();  
     }  
 }  
+```
   
-```  
+```vb
```
Comment (mairaw) [inline xml/System.Net.Http/HttpClient.xml]: same on line 61. do you want to add there as well?

**Comment by Youssef1313** [comment]
@mairaw , My forked repo was deleted and couldn't push more commits to this PR. I'll open a new one and then close this.

**Comment by Youssef1313** [comment]
https://github.com/dotnet/dotnet-api-docs/pull/2936

Closing this..

**Pull request closed without merging**